requirements/test.txt

```diff
@@ -1,7 +1,8 @@
-coverage>=5.0
+coverage>=5.2
```

Comment:
do we still need this? isn't `pytest-cov` enough?

Reply:
well, it is installed under the hood anyway :]

Reply:
it would be nice to prune extra dependencies, even if they are installed under the hood

Reply:
I see your point, but that could be a bit tricky as we have many cached in CI so removing some user may found it is needed in the end...
btw, even in the pytest extension, they suggest to pin `coverage>=5.2`